grails-app/assets/javascripts/organisation.js

```diff
@@ -106,7 +232,7 @@ OrganisationViewModel = function (props, options) {
         if ($('.validationEngineContainer').validationEngine('validate')) {
 
             var orgData = self.modelAsJSON(true);
-            $.ajax(config.organisationSaveUrl, {type:'POST', data:orgData, contentType:'application/json'}).done( function(data) {
+            $.ajax(fcConfig.organisationSaveUrl, {type:'POST', data:orgData, contentType:'application/json'}).done( function(data) {
```

Comment:
We don't what to code explicit dependencies on fcConfig.  If the option was missing pass it to the contructor of the view model.